src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -436,7 +436,7 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
             </View>
           </GenericSectionItem>
         </Section>
-        {inspectableTokenWarningText && (
+        {inspectableTokenWarningText && !phoneNumber && (
```

Comment:
is phoneNumber at that point valid? I mean existing customer? or is just non-empty number.

Reply:
if `on-behalf-of` toggle is enabled, then `phoneNumber` will have a value from API and is passed on from the previous screen.

Otherwise if `on-behalf-of` toggle is disabled, then `phoneNumber` will be undefined.